### DISCUSSION TITLE
[QUESTION] no profile were found

### DISCUSSION THREAD

**Discussion by vb-ben** [fastlane/fastlane#17372] [Q&A]
Hello everybody,

i have a bug but i do not know how to say, i do not understand how fastlane work , i read document and implement into my project but not work.

❌  error: No profiles for 'xxx' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'xxx'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'xxx' from project 'xxx')

<img width="1425" alt="Screen Shot 2020-10-05 at 13 52 54" src="https://user-images.githubusercontent.com/5819750/95048454-1353d900-0712-11eb-8cd3-8afec91609cb.png">

** ARCHIVE FAILED **
[13:52:20]: Exit status: 65

+-------------+-------------------------+
|           Build environment                           |
+-------------+-------------------------+
| xcode_path  | /Applications/Xcode.app  |
| gym_version | 2.162.0                               |
| sdk                | iPhoneOS14.0.sdk            |
+-------------+-------------------------+

<img width="1429" alt="Screen Shot 2020-10-05 at 13 55 51" src="https://user-images.githubusercontent.com/5819750/95048689-7cd3e780-0712-11eb-99fc-553ad1c6e385.png">

Any solution, thanks much

**Comment by rogerluan** [comment]
That's a code signing issue. You're not providing `xcodebuild` a valid provisioning profile to sign your build, and automatic code signing is not enabled. You can either pass a valid profile to xcodebuild, or let it auto-sign for you, as the error log states:

>To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild
